pkgs/development/interpreters/metamath/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  name = "metamath-${version}";
+  version = "0.167";
+
+  buildInputs = [ autoreconfHook ];
+
+  src = fetchFromGitHub {
+    owner = "Taneb";
+    repo = "metamath";
+    rev = "579166a649c5d9ae0cdc641185e68219d2812f24";
+    sha256 = "0ly7qv3gajlaf283fsy9ak41r09sh49ygzj0s9yxyjfli6rgxdby";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Interpreter for the metamath proof language";
+    longDescription = ''
+      The metamath program is an ASCII-based ANSI C program with a command-line
+      interface. It was used (along with mmj2) to build and verify the proofs
+      in the Metamath Proof Explorer, and it generated its web pages. The *.mm
+      ASCII databases (set.mm and others) are also included in this derivation.
+      '';
+    homepage = "https://us.metamath.org";
+    downloadPage = "http://us.metamath.org/downloads/metamath.tar.gz";
```

It doesn't look like download "page."
```suggestion
    downloadPage = "http://us.metamath.org/#downloads";
```